innovation.game.php

```diff
@@ -15834,6 +15836,7 @@ function stInterInteractionStep() {
                     // Return revealed cards back to player's hand.
                     $cards = self::getCardsInLocation($player_id, 'revealed');
                     foreach ($cards as $card) {
+                        $card = self::getCardInfo($card['id']);
```

Comment:
This bug doesn't actually happen when the card is in a publicly visible location (e.g. revealed or board). So let's remove this line so that we aren't making an unnecessary SQL query.